src/metabase/automagic_dashboards/core.clj

```diff
@@ -243,11 +245,15 @@
 (defn- source
   [card]
   (cond
+    ;; This is a non-native question with no ancestors (It is a query on a table).
```

Comment:
I don't understand this comment. The only thing checked here is if the card is a model. Non-native, no ancestors, even "query on a table" can all be invalidated. I'm not seeing anything the comment really could say and this feels wrong.

Reply:
Yeah, that might be historical and have gotten shifted around. Updating to simply "This is a model".